### DISCUSSION TITLE
docs/v1.10.x/advance/offload-mellanox/

### DISCUSSION THREAD

**Discussion by giscus** [kubeovn/kube-ovn#1835] [Announcements]
# docs/v1.10.x/advance/offload-mellanox/

https://kubeovn.github.io/docs/v1.10.x/advance/offload-mellanox/

**Comment by iceCreeam** [comment]
是否考虑卸载ovn控制面和ovs转发面到bf2的arm核中？类似bf2和openshift做的这种方案，提供一种标准化接口，以对接各家厂商DPU？
